runtime/client_http_request.go

```diff
@@ -79,6 +81,7 @@ func NewClientHTTPRequest(
 		ContextLogger:        NewContextLogger(client.loggers[clientMethod]),
 		defaultHeaders:       client.DefaultHeaders,
 		ctx:                  ctx,
+		jsonWrapper:          client.JSONWrapper,
```

Comment:
something like this reads better: `jsonMashaller: json.Marshaller` so you'll need to update the package name, field name and interface name